package.json

```diff
@@ -33,6 +33,7 @@
     "eslint": ">= 6.x"
   },
   "devDependencies": {
+    "@auto-it/all-contributors": "^9.53.1",
```

Comment:
Can you update the version of `auto` to match this too?

Reply:
Hi @hipstersmoothie 

I am sorry but I am having trouble understanding what I need to do here.
Do I run `yarn upgrade` or `npm update` on `auto`?

Running `yarn outdated` showed that there is a much newer version of `auto` available.

Kindly let me know what needs to be done.
Thanks!

Reply:
Can you bump both to the *same* version

Reply:
Hello @kendallgassner 
Does this mean the required change is as follows?

In `package.json`
```
"devDependencies": {
    "@auto-it/all-contributors": "^9.53.1",
    ...
    "auto": "^9.53.1",
    ...
}
```

Reply:
yes! thats perfect